frontend/server/src/Controllers/Identity.php

```diff
@@ -324,31 +324,39 @@ public static function apiBulkCreateForTeams(\OmegaUp\Request $r): array {
         }
         /** @var array<string, bool> $seenUsernames */
         $seenUsernames = [];
-        foreach ($teamIdentities as $identity) {
-            if (isset($seenUsernames[$identity['username']])) {
+        /** @var array<string, bool> $seenMemberUsernames */
+        $seenMemberUsernames = [];
+        foreach ($teamIdentities as $username => $teamIdentity) {
+            if (isset($seenUsernames[$teamIdentity['username']])) {
                 throw new \OmegaUp\Exceptions\DuplicatedEntryInDatabaseException(
                     'teamAliasInUse'
                 );
             }
-            $seenUsernames[$identity['username']] = true;
+            $seenUsernames[$teamIdentity['username']] = true;
+
+            // When usernames index is provided we need to avoid duplicated
+            // users in different teams
```

Comment:
se puede mover esto a arriba de L343? me confundió un poco el hecho de que esto está arriba del código que checa el caso de cuando _no_ se proveen los usuarios. también, el csv no tiene índices, así que esto sería un poquitín más correcto.

```suggestion
            // When usernames are provided we need to avoid duplicated
            // users in different teams.
```

Reply:
Listo.